lib/expand.js

```diff
@@ -313,15 +332,23 @@ api.expand = ({
       continue;
     }
 
-    const container = _getContextValue(activeCtx, key, '@container') || [];
+    // use potential scoped context for key
+    let termCtx = activeCtx;
+    const ctx = _getContextValue(activeCtx, key, '@context');
+    if(ctx) {
+      termCtx = _processContext(
+        {activeCtx, localCtx: ctx, options});
```

Comment:
Seems like this param would fit on the same line as `_processContext` (i.e. no need to drop it down).